tests/openvino/native/quantization/test_weights_compression.py

```diff
@@ -1383,6 +1420,14 @@ def test_data_aware_algo_with_different_activation_dimensions(n_extra_dims):
     )
 
 
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        LMLinearModel.INPUT_SHAPE,
+        [3, 5, 16],
+        [1, 1, 16],
```

Comment:
Added test case for the bug 155538 with tiny-llama example. The root cause is not filtered data with sequence length == 1.
@kshpv